### PR TITLE
Use `sentry_check_cli_installed` to verify Sentry's CLI requirements

### PR DESCRIPTION
**Pull request by mokagio**
This is an action [provided by the Sentry Fastlane plugin](https://github.com/getsentry/sentry-fastlane-plugin/tree/3dff83c640a4ebb92beb74e6122cccd5e5e77429#checking-the-sentry-cli-is-installed) that verifies the CLI is installed and its version is compatible with the plugin's expectation.

_Opening against the release branch because we're more likely to run lanes using this check from here than from `trunk`._

## Extra context

I recently had to deploy some builds locally. The first time I tried, the process failed because my `sentry-cli` version was too old. We already had a custom check to ensure that the CLI is installed at the start of the lane, but that didn't cover the version. It was an annoying failure because it occurred after my machine was busy building and uploading... This should make us fail ASAP if the version is not the expected one, avoiding work that's doomed to be thrown away.

I went in search for this after bumping into a Sentry-related change in WooCommerce: https://github.com/woocommerce/woocommerce-ios/pull/6044#discussion_r800702937.

## Testing

This is the action output in the happy path:

image

If the CLI is missing, the lane will fail with the following output:

image

I started looking into how to install an older version of the CLI to verify that path, too, but it proved to be a fiddly process. I gave up in the interest of not getting sidetracked, trusting that the plugin does what it's supposed to.

## Regression Notes
1. Potential unintended areas of impact
N.A.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N.A.

3. What automated tests I added (or what prevented me from doing so)
N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

File: fastlane/Fastfile
Comment (AliSoftware) [inline fastlane/Fastfile:1035]: That's quite a nice and welcome change :)

At that point though I'm not sure it's worth keeping a `def` dedicated method that ends up just calling the action though, as opposed to calling `sentry_check_cli_installed` directly everywhere we previously called `ensure_sentry_installed` in the rest of the `Fastfile`?

**Review by AliSoftware — COMMENTED** [review]
It happens that I have an older version of `sentry-cli` myself, so that was the perfect occasion to test the remaining case that you were not able to test yourself:

```
$  sentry-cli --version
sentry-cli 1.58.0

$ fl build_and_upload_installable_build
…
+------------------+------------------------------------+
|                     Lane Context                      |
+------------------+------------------------------------+
| DEFAULT_PLATFORM | ios                                |
| PLATFORM_NAME    |                                    |
| LANE_NAME        | build_and_upload_installable_build |
+------------------+------------------------------------+
[16:02:53]: Your sentry-cli is outdated, please upgrade to at least version 1.70.1 and start your lane again!
```
So all working as expected 👍 


I'd still suggest to replace the 3 calls to `ensure_sentry_installed` with a direct call to `sentry_check_cli_installed` instead, and remove the `def` helper function which is not really relevant anymore at that point (our `Fastfile`s are already long enough as it is, a quick cleanup is always welcome, however small 🙂 .

**Comment by mokagio** [comment]
> I'd still suggest to replace the 3 calls to `ensure_sentry_installed` with a direct call to `sentry_check_cli_installed` instead

I considered that, actually.

It was so handy to make the change in one single place, though, that I decided to leave it as is _just in case_.

Of course, one could argue that we're unlikely to change the implementation now that we're relying on Sentry itself for this... 🤷‍♂️⚖️

Let me know if you feel strongly about this and I'll address 👌

**Comment by mokagio** [comment]
Also, thanks for checking with the outdated version.

What a lucky coincidence 😁😁

**Comment by AliSoftware** [comment]
> Let me know if you feel strongly about this and I'll address 👌

Well my thoughts would be YAGNI hence the suggestion for cleanup 🙃  but 🤷  up to you 🙂

**Review by AliSoftware — APPROVED** [review]
Approving to unblock, and will let you decide if you want to apply my decision before merging or not 🙂 

[EDIT] Eh, well, forgot about auto-merge… well that settles it then 😂

**Pull request merged**
